Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -169,9 +173,16 @@ private void OnLoaded(object sender, RoutedEventArgs e)
             {
                 _stateGroup.CurrentStateChanged += OnVisualStateChanged;
                 _narrowState = GetTemplateChild(NarrowState) as VisualState;
-                UpdateView(true);
             }
 
+            _selectionStateGroup = (VisualStateGroup)GetTemplateChild(SelectionStates);
+            if (_selectionStateGroup != null)
+            {
+                _selectionStateGroup.CurrentStateChanged += OnSelectionStateChanged;
```

Comment:
Why do we need an event handler for selection state changing? We already know when selection changes, and we know when the width state changes